src/redux/features/observations/utils.ts

```diff
@@ -17,7 +17,7 @@ import { regexMatch } from 'src/utils/search';
 
 // utils
 
-export const searchResultPlots = (search: string, zone?: ObservationPlantingZoneResults) => {
+export const searchResultPlots = (search: string, zone: ObservationPlantingZoneResults | null) => {
```

Comment:
Is this just to be more explicit about the result here? That is, would it have worked the same before this change?